harry-olabs/avi-hub
# AVI-21: Implement the users management page

### What does this PR do


This is the users management page implemented with react.

Closes #21

### How should it be tested



- Spin up the app in dev mode with `npm start`
- Navigate to the Manage Users page
- App should run fine with smooth navigation to pages


### Screenshots



| Before | After |
|----------------|----------------|
|  <img width="1381" alt="Screenshot 2023-08-02 at 11 58 03" src="https://github.com/harry-olabs/avi-hub/assets/135703492/132d9e95-2125-4c01-a5d5-13fc11a74327"> | ![Screenshot (153)](https://github.com/harry-olabs/avi-hub/assets/80858126/a7272bf2-22d3-43da-8262-343be9c8d6da)
